components/Breadcrumb.vue

```diff
@@ -45,60 +45,77 @@
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
+import { getKeycloakRoles } from '@/utils'
 
 @Component({})
 export default class Breacrumb extends Vue {
-  get backDisabled(): boolean {
-    if ((this.$route.path === '/dashboard') || (this.$route.path === '/')) {
-      return true
+  /** Whether the back button should be disabled. */
+  get isBackDisabled(): boolean {
+    switch (this.$route.path) {
+      case '/':
+      case '/dashboard':
+      case '/dashboard/': return true
     }
     return false
   }
 
+  /** The list of breadcrumbs for the current route. */
   get breadcrumbs(): Array<any> {
-    if ((this.$route.path === '/ppr-marketing') || (this.$route.path === '/ppr-marketing/')) {
-      return [
-        {
-          disabled: false,
-          href: '/',
-          text: 'BC Registries and Online Services',
-        },
-        {
-          disabled: true,
-          href: '',
-          text: 'Personal Property Registry',
-        },
-      ]
-    }
-
-    if ((this.$route.path === '/dashboard') || (this.$route.path === '/dashboard/')) {
-      return [
-        {
-          disabled: true,
-          href: '',
-          text: 'BC Registries Dashboard',
-        },
-      ]
-    }
-
-    if (this.$route.path === '/') {
-      return [
-        {
-          disabled: true,
-          href: '',
-          text: 'BC Registries and Online Services',
-        },
-      ]
+    switch (this.$route.path) {
+      case '/ppr-marketing':
+      case '/ppr-marketing/': {
+        return [
+          {
+            disabled: false,
+            href: '/',
+            text: 'BC Registries and Online Services',
+          },
+          {
+            disabled: true,
+            href: '',
+            text: 'Personal Property Registry',
+          },
+        ]
+      }
+
+      case '/dashboard':
+      case '/dashboard/': {
+        let isStaff: boolean
+        try {
+          const keycloakRoles = getKeycloakRoles()
```

Comment:
I reinstated this functionality after UX changed their mind on the text.

Note the catch block below in case `getKeycloakRoles()` throws an error.